src/services/exports/exports.repository.ts

```diff
@@ -271,9 +271,9 @@ const generateCsvStructure = async (
           .Model.countDocuments({
             'structure.$id': new ObjectId(structure._id),
           });
-        const user: IUser = await app
+        const users: IUser[] = await app
           .service(service.users)
-          .Model.findOne({ 'entity.$id': new ObjectId(structure._id) });
+          .Model.find({ 'entity.$id': new ObjectId(structure._id) });
```

Comment:
est ce que ca vaut le coud de rajouter un index là dessus ?

Reply:
normalement c'est déjà fait pour le entity.$id (A confirmer)

Reply:
apparemment il n'existe pas

Reply:
"You can use the .$id reference but it will ignore any indexes on those fields"